### PR TITLE
Fixes for suit sensors and medical records console

### LINKED ISSUE #10132: [BUG] medical console "bip" spam
### There is no other issue reporting the same problem.
- [X] I have searched the existing issues

### Game Version
23.10.11.00

### I was
Server Local Host
HOS
SquareStation

### OS
**OS:** Lubuntu 22.04x64 
**Kernel:** 5.15.0-86-generic 
**Memory**: 16Gio Ram
**CPU**: Intel i7-6700HQ (8) @ 3.500GHz
**GPU**: NVIDIA GeForce GTX 960M 

### What happened?

Hi, I've found a big, dirty bug.
I was on a localhost session to test some uniforms.
- I spawned a lot of characters with the F7 key
- I deleted the two terminal medical (because of too much noise )

beeping can be heard throughout the station
without attenuation, and continuously
I did spawn a few characters with the F7 key  
Capture d’écran du 2023-10-13 08-34-16

### LINKED ISSUE #10133: [BUG]  Suit sensors aren't synced properly
### There is no other issue reporting the same problem.

- [X] I have searched the existing issues

### Game Version

23101306

### I was

Client

### OS

Windows 8, 10, 11

### What happened?

image

- Suit sensors no longer report your name within the medical records console
- It will also fail to sync the correct mode of the suit sensor

### What should have happened instead?

Report the name and the correct info

### Steps to reproduce the issue

1. Go to a medical records console
2. Change your suit sensors mode
3. profit

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MaxIsJoe**
fix #10133
fix #10132

CL: [Improvement] Added some feedback text to turning on or off suit sensors.
CL: [Improvement] Added some feedback text to turning on or off the alert sound of the medical record console.
CL: [Fix] Fixed issue with medical records terminal not grabbing the name of the reported patient.
CL: [Fix] Reduced the distance of the medical records alert sound.
CL: [Fix] Fixed several elements in suit sensors not getting synced correctly to the server and clients.
CL: [Fix] Fixed an issue with the medical records console not syncing its alert mute function.
CL: [Fix] Fixed an exploit that would result in hacked clients remotely changing other crew member's suit sensor settings.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/SuitSensor.cs [hunk 1]
```diff
@@ -144,34 +153,46 @@ public string GetInfo()
...
-
 		}
 
-		private void SwitchMode()
+		[Command(requiresAuthority = false)]
+		private void SwitchMode(PlayerScript player)
 		{
+			if (Vector3.Distance(gameObject.AssumedWorldPosServer(), player.AssumedWorldPos) > 3.5)
+			{
+				Loggy.LogWarning("[MedicalTerminal/SwitchMode] - Prevented possible cheating from player who is far away from this option.");
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/SuitSensor.cs]: Log the player?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/SuitSensor.cs]: Good point, will add it when im back home

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/SuitSensor.cs]: Done

**Comment by MaxIsJoe** [comment]
approvals pls

**Review by corp-0 — APPROVED** [review]

**Pull request merged**